src/uwtools/drivers/forecast.py

```diff
@@ -184,17 +116,10 @@ def run(self, cycle: datetime) -> bool:
             print(full_cmd, file=sys.stdout)
             return True
 
-        result = execute(cmd=full_cmd)
+        with change_dir(run_directory):
+            result = execute(cmd=full_cmd)
```

Comment:
The executables should typically be run from the directory where all the data is staged.

Reply:
Yep! I can make that change. I think `execute()` came around in between me doing this originally for MPAS and now and I didn't make that connection.